<launch>
  <arg name="camera_info" default="$(find uav_monocular_nav)/config/camera_info.yaml" />
  <node pkg="uav_monocular_nav" type="inference_node.py" name="inference_node" output="screen">
    <param name="camera_info" value="$(arg camera_info)" />
    <param name="segmentation_model" value="$(find uav_monocular_nav)/training/segmentation.pth" />
    <param name="planner_model" value="$(find uav_monocular_nav)/training/planner.pth" />
  </node>
  <node pkg="uav_monocular_nav" type="simple_uav_controller.py" name="simple_uav_controller" output="screen" />
</launch>
